app/frontend/actions.js

```diff
@@ -233,13 +236,15 @@ module.exports = ({setState, getState}) => {
     const balance = await wallet.getBalance()
     const ownAddressesWithMeta = await wallet.getOwnAddressesWithMeta()
     const transactionHistory = await wallet.getHistory()
+    const conversionRates = await getConversionRates()
```

Comment:
This is probably a bit dangerous since you don't catch possible exceptions if fetching of the rates fails. I would wrap this into a try-catch block and if fetching fails, I would set the rates to null. If the rates are null, they would not simply display, since it is not critical.

Reply:
Yep. We even might want to fetch them in parallel so that slow request doesn't block the main wallet